controllers/runtime.go

```diff
@@ -144,6 +144,15 @@ func retryGetPushSecretName(r *SpecialResourceReconciler) (string, error) {
 
 func getPushSecretName(r *SpecialResourceReconciler) (string, error) {
```

Comment:
in vanilla k8s there are no push secrets?

Reply:
I don't think there are. At least not in my kOps cluster